.github/workflows/code-review.yml

```diff
@@ -0,0 +1,64 @@
+name: Code Review
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize]
+    paths:
+      - "*"
+      - "!infra/*"
+      - "!.github/*"
+
+jobs:
+  js_code_review:
+    runs-on: ubuntu-22.04
+
+    steps:
```

Comment:
is missing a step to run the integration test.
Take a look in [here](https://github.com/pagopa/io-functions-services/blob/master/.devops/code-review-pipelines.yml#L81)